packages/legend-application-studio/src/components/editor/editor-group/service-editor/ServiceEditor.tsx

```diff
@@ -285,157 +359,348 @@ const ServiceGeneralEditor = observer(() => {
         registration"
           update={toggleAutoActivateUpdates}
         />
-        <div className="panel__content__form__section">
-          <div className="panel__content__form__section__header__label">
-            Owners
-          </div>
-          <div className="panel__content__form__section__header__prompt">
-            {`Specifies who can manage and operate the service (requires minimum ${MINIMUM_SERVICE_OWNERS}
-          owners).`}
-          </div>
-          <div className="panel__content__form__section__list">
-            <div
-              className="panel__content__form__section__list__items"
-              data-testid={
-                LEGEND_STUDIO_TEST_ID.PANEL_CONTENT_FORM_SECTION_LIST_ITEMS
-              }
-            >
-              {owners.map((value, idx) => (
-                <div
-                  key={value}
-                  className={
-                    showOwnerEditInput === idx
-                      ? 'panel__content__form__section__list__new-item'
-                      : 'panel__content__form__section__list__item'
-                  }
-                >
-                  {showOwnerEditInput === idx ? (
-                    <>
-                      <input
-                        className="panel__content__form__section__input panel__content__form__section__list__new-item__input"
-                        spellCheck={false}
-                        disabled={isReadOnly}
-                        value={ownerInputValue}
-                        onChange={changeOwnerInputValue}
-                      />
-                      <div className="panel__content__form__section__list__new-item__actions">
-                        <button
-                          className="panel__content__form__section__list__new-item__add-btn btn btn--dark"
-                          disabled={
-                            isReadOnly || owners.includes(ownerInputValue)
+        {owners.length == 0 && (
```

Comment:
<img width="312" alt="Screenshot 2023-09-18 at 3 38 54 PM" src="https://github.com/finos/legend-studio/assets/73408381/ded611d8-a7f2-4d66-b6c6-fb09c2fe8068">
could we wrap up the code for owners so the space between `owners` and `Activate` will be kept